env/base-sepolia.json

```diff
@@ -2,25 +2,20 @@
   "network": {
     "chainId": 84532,
     "environment": "testnet",
-    "centrifugeId": 2
+    "centrifugeId": 2,
+    "connectsTo": [
+      "sepolia"
+    ]
   },
   "adapters": {
     "wormhole": {
-      "deploy": true,
-      "relayer": "0x7B1bD7a6b4E61c2a123AC6BC2cbfC614437D0470",
-      "chain-id": 10004,
-      "connectsTo": [
-        "sepolia"
-      ]
+      "wormholeId": 10004,
+      "relayer": "0x93BAD53DDfB6132b0aC8E37f6029163E63372cEE"
```

Comment:
Relayer was off, should be the one from https://wormhole.com/docs/build/reference/contract-addresses/#__tabbed_3_2